files/fr/web/api/css_font_loading_api/index.html

```diff
@@ -0,0 +1,48 @@
+---
+title: CSS Font Loading API
+slug: Web/API/CSS_Font_Loading_API
+tags:
+  - API
+  - CSSFontLoading
+  - CSSOM
+  - Experimental
+  - Fonts
+  - NeedsContent
+  - Reference
```

Comment:
tags can be removed in the localized page :) There are a relique of the past and are not localized nor even visible somehow to the end user.

Reply:
good to know!